stf/build.gradle.kts

```diff
@@ -13,12 +13,14 @@ sarosEclipse {
     isAddDependencies = true
     pluginVersionQualifier = versionQualifier
     eclipseVersion = eclipseVersionNr
+    configs = listOf("bundle", "bundleApi")
```

Comment:
With your last commit you defined the exact same list as default configs. Therefore, I think this configuration is redundant.

Reply:
Oh yes, we missed that one. We already removed it accordingly.